tests/5.0/target/test_target_device1.c

```diff
@@ -0,0 +1,53 @@
+//===------test_target_device.c--------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Description:
+// This is a basic test to demonstrate target + parallel for + device clause
+//
+//===------------------------------------------------------------------------===//
+#include <stdio.h>
+#include "omp.h"
+#include "ompvv.h"
+
+#define THREADS 1024*1024*1024
```

Comment:
The macro name THREADS should probably be changed to N, since it does not determine the number of threads that will be created

Reply:
Replaced 'THREADS' with 'N'